### PR TITLE
Testing/etais

### PR DESCRIPTION
**Pull request by sndvsk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
-   a. Waldur Organization = Get or create organization based on project affiliation data from EOSC MP
-
-   b. Waldur Project = get or create project based on project data
-
-   c. Waldur Order = create a new order for the offering using info from EOSC MP
-
-   d. Add new members into the project if not yet present based on the info about the request.
+##Environmental variables
+### utils_orders.py
+EOSC_URL -  \
```
Comment (sergei-zaiaev) [inline README.md]: Could you please add some description for each of env. vars?

File: src/utils_offers.py [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+
+
+def get_offer_list_of_resource(resource_id):
+    headers = resource_and_offering_request()
+    response = requests.get(urllib.parse.urljoin(EOSC_URL, OFFER_LIST_URL % (str(resource_id))),
+                            headers=headers)
+    if response.status_code == 200:
+        return response.json()
+    else:
+        raise
```
Comment (sergei-zaiaev) [inline src/utils_offers.py]: `raise` what? Could you please make this more specific?

File: src/utils_orders.py [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+        if property_name == item['id']:
+            return item['value']
+
+
+def get_plan(eosc_project_item_data, waldur_offering_data):
+    plan = None
+    for waldur_plan in waldur_offering_data['plans']:
+        if eosc_project_item_data.attributes.offer == waldur_plan['name']:
+            plan = waldur_plan
+    # if plan is None:
```
Comment (sergei-zaiaev) [inline src/utils_orders.py]: Do we still need this comment?

File: src/utils_orders.py [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+PASSWORD = os.environ.get('PASSWORD')
+WALDUR_URL = os.environ.get('WALDUR_URL')
+WALDUR_API = urllib.parse.urljoin(WALDUR_URL, 'api/')
+WALDUR_API_AUTH = urllib.parse.urljoin(WALDUR_URL, 'api-auth/password/')
+
+WALDUR_TARGET_ORGANIZATION_NAME = os.environ.get('WALDUR_TARGET_ORGANIZATION_NAME', 'Test4All')
+
+WALDUR_TOKEN = os.environ.get('WALDUR_TOKEN')
+
+# def get_waldur_token(): #
```
Comment (sergei-zaiaev) [inline src/utils_orders.py:29]: Do we still need these functions?

Comment (sndvsk) [reply src/utils_orders.py:29]: will keep 29-34 function to test with local waldur

**Pull request merged**